goal_src/jak2/build/all_objs.json

```diff
@@ -0,0 +1,2252 @@
+["mood-funcs2",                                "mood-funcs2",                3, ["ENGINE", "GAME"], "engine/gfx/time-of-day"],
+["weather-part",                               "weather-part",               3, ["ENGINE", "GAME"], "engine/gfx/time-of-day"],
+["time-of-day",                                "time-of-day",                3, ["ENGINE", "GAME"], "engine/gfx/time-of-day"],
+["sky-data",                                   "sky-data",                   3, ["ENGINE", "GAME"], "engine/data"],
+["sky-tng",                                    "sky-tng",                    3, ["ENGINE", "GAME"], "engine/gfx/sky"],
+["load-state",                                 "load-state",                 3, ["ENGINE", "GAME"], "engine/load"],
+["level-info",                                 "level-info",                 3, ["ENGINE", "GAME"], "engine/level"],
+["level",                                      "level",                      3, ["ENGINE", "GAME"], "engine/level"],
+["text",                                       "text",                       3, ["ENGINE", "GAME"], "engine/ui"],
+["collide-hash",                               "collide-hash",               3, ["ENGINE", "GAME"], "engine/collide"],
+["collide-probe",                              "collide-probe",              3, ["ENGINE", "GAME"], "engine/collide"],
+["collide-frag",                               "collide-frag",               3, ["ENGINE", "GAME"], "engine/collide"],
+["collide-mesh",                               "collide-mesh",               3, ["ENGINE", "GAME"], "engine/collide"],
+["collide-touch",                              "collide-touch",              3, ["ENGINE", "GAME"], "engine/collide"],
+["collide-edge-grab",                          "collide-edge-grab",          3, ["ENGINE", "GAME"], "engine/collide"],
+["collide-shape",                              "collide-shape",              3, ["ENGINE", "GAME"], "engine/collide"],
+["collide-shape-rider",                        "collide-shape-rider",        3, ["ENGINE", "GAME"], "engine/collide"],
+["collide",                                    "collide",                    3, ["ENGINE", "GAME"], "engine/collide"],
+["collide-planes",                             "collide-planes",             3, ["ENGINE", "GAME"], "engine/collide"],
+["spatial-hash",                               "spatial-hash",               3, ["ENGINE", "GAME"], "engine/data"],
+["actor-hash",                                 "actor-hash",                 3, ["ENGINE", "GAME"], "engine/data"],
+["merc-death",                                 "merc-death",                 3, ["ENGINE", "GAME"], "engine/gfx/merc"],
+["water-flow",                                 "water-flow",                 3, ["ENGINE", "GAME"], "engine/gfx/water"],
+["water-h",                                    "water-h",                    3, ["ENGINE", "GAME"], "engine/gfx/water"],
```

Comment:
these two water files aren't about graphics - they are about in the in-game water objects.  Maybe these could go in a `engine/common_objs` or something like that. Or even `process-drawable` - the `water-control` type is one of the things that makes up a `process-drawable`, and the `water-vol` is a specialization of `process-drawable.`